src/test/components/legal-information/service-header/index.njk

```diff
@@ -0,0 +1,6 @@
+<html>
+  <body>
+      {% from "../../../../../src/hmpps/components/legal-information/service-header/macro.njk" import serviceHeader %}
```

Comment:
would be nice for this to mimic how a service would use this?
`hmpps/components/legal-information/service-header/macro.njk` kind of path (might need to tweak the nunjucks configure in the test?) https://github.com/ministryofjustice/hmpps-custody-manager-ui/blob/main/server/views/partials/layout.njk#L2 is the example in custody manager